docs/advanced-features/tabulator.md

```diff
@@ -86,10 +114,7 @@ In addition to the columns defined in the schema, Tabulator will add:
 3. **Cost Management**: Querying very large datasets can be expensive
    (approximately dollars per terabyte). Be sure to set up appropriate cost
    controls and monitoring.
-4. **Access Restrictions**: Due to the way permissions are configured, Tabulator
-   cannot be accessed from the AWS Console or views. You must access Tabulator
```

Comment:
it still cannot be accessed from the AWS Console or views

Reply:
Right, I say that in the previous section. Is that okay?

Reply:
ah, okay